### PR TITLE
Feature/86 announcement banner

### PR DESCRIPTION
**Pull request by timohuber**

### CODE REVIEW

File: pool/app/announcement/announcement.ml [hunk 1]
```diff
@@ -0,0 +1,9 @@
+include Entity
+include Event
+include Entity_guard
+
+let find = Repo.find
+let all = Repo.all
+let find_admin = Repo.find_admin
+let find_by_user = Repo.find_by_user
+let find_of_tenant = Repo.find_of_tenant
```
Comment (mabiede) [inline pool/app/announcement/announcement.ml:9]: As we anyway have interface files, we can simplify this
```suggestion
include Repo
```

File: pool/app/pool_database/migrations/migration_202309211305.ml [hunk 2]
```diff
@@ -95,6 +95,7 @@ let add_default_guardian_role_permission_root =
     {sql|
     INSERT INTO `guardian_role_permissions` (`role`, `permission`, `target_model`) VALUES
     ('`Operator', 'manage', '`Admin'),
+    ('`Operator', 'manage', '`Announcement'),
```
Comment (mabiede) [inline pool/app/pool_database/migrations/migration_202309211305.ml:98]: Please add separate migration

File: pool/routes/routes.ml [hunk 3]
```diff
@@ -116,7 +123,7 @@ module Public = struct
               [ CustomMiddleware.Guardian.require_user_type_of
                   Pool_context.UserType.[ Contact; Admin ]
               ]
-            [ get "/logout" Login.logout ]
+            [ get "/logout" Login.logout; announcements ]
```
Comment (mabiede) [inline pool/routes/routes.ml:126]: Should announcements be public?

File: resources/index.scss [hunk 4]
```diff
@@ -583,3 +583,23 @@ $n: 12;
 .break-grid {
     grid-column: 1;
 }
+
+
+// ANNOUNCEMENT
```
Comment (mabiede) [inline resources/index.scss:588]: caps

File: pool/routes/routes.ml (hunk 3 above)
Comment (timohuber) [reply pool/routes/routes.ml:126]: No, the issue states that the banner should be shown to contacts and/or admins

**Review by mabiede — APPROVED** [review]

**Pull request merged**